packages/broker/src/plugins/operator/MaintainTopologyHelper.ts

```diff
@@ -125,29 +124,28 @@ export class MaintainTopologyHelper extends EventEmitter<MaintainTopologyHelperE
         }
         const parseItems = (response: any) => {
             if (!response.operator) {
-                this.logger.error(`Operator ${this.address.toLowerCase()} not found in TheGraph`)
+                logger.debug('Unable to find operator in The Graph', { operatorContractAddress: this.operatorContractAddress })
```

Comment:
is this not an error?

Reply:
Good point. I guess this is somewhat of a situation that would warrant user intervention as described here: https://github.com/streamr-dev/network/blob/main/internal-docs/logging-practices.md#use-log-levels-appropriately. I'll change it back to error.